adops/setting-up-prebid-for-amp-in-dfp.md

```diff
@@ -34,64 +34,53 @@ In addition to your other line item settings, you'll need the following:
 
 + Set **Rotate creatives** to *Evenly*.
 
-+ In the targeting section, select **Key-values** targeting.  In [Show Prebid Ads on AMP Pages]({{site.github.url}}/dev-docs/show-prebid-ads-on-amp-pages.html), we targeted the "prebid_amp" keyword set to "true" from the developer side as an example, but you will want to coordinate with your development team to use your own key-values.
++ In the targeting section, select **Key-values** targeting.  You'll need to coordinate with your development team on what key-values you want to target.
 
 Save your line item and add a creative.
 
-
 ## Creative Setup
 
 On the new creative screen, select the **Third party** creative type.
 
-Enter the below code snippet in the **Code snippet** text area, and make sure to uncheck the **Serve into a SafeFrame** checkbox.
+Enter the below code snippet in the **Code snippet** text area.
+
+{: .alert.alert-success :}
+You can always get the latest version of the creative code below from [the AMP example creative file in our GitHub repo](https://github.com/prebid/prebid-universal-creative/blob/master/template/amp/dfp-creative.js).
 
-Note that you can always get the latest version of the creative code below from [the AMP example creative file in our GitHub repo](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/amp/creative.html).
+{% highlight html %}
 
-```html
-<!-- This script tag should be returned by your ad server -->
+    <script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative/dist/creative.js"></script>
```

Comment:
@rmloveland Just found out from https://www.jsdelivr.com/features website that omitting version from url is not recommended for production. 
Please update the src to  `https://cdn.jsdelivr.net/npm/prebid-universal-creative@0.1.2/dist/creative.js`

Reply:
Thanks, updated to use the new source URL in 59fadd3